src/main/java/trade/wayruha/mexc/dto/Order.java

```diff
@@ -1,33 +1,70 @@
 package trade.wayruha.mexc.dto;
 
+import com.fasterxml.jackson.annotation.JsonAlias;
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-import lombok.Data;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
 import trade.wayruha.mexc.enums.OrderSide;
 import trade.wayruha.mexc.enums.OrderStatus;
 import trade.wayruha.mexc.enums.OrderType;
 import trade.wayruha.mexc.enums.TimeInForce;
 
 import java.math.BigDecimal;
 
-@Data
+@Getter
+@Setter
+@RequiredArgsConstructor
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class Order {
-    private String symbol;
-    private long orderId;
+public class Order extends WSBaseDto {
```

Comment:
This Order dto is not related to web sockets. why do we extend it from WSBaseDto?

Reply:
It's related, see method  **trade.wayruha.mexc.ws.WSClientFactory#userSpotTradesSubscription**